seed/helpers_test.go

```diff
@@ -47,16 +50,12 @@ func (s *helpersSuite) SetUpTest(c *C) {
 	s.BaseTest.SetUpTest(c)
 	s.BaseTest.AddCleanup(snap.MockSanitizePlugsSlots(func(snapInfo *snap.Info) {}))
 
-	s.TestingSeed = &seedtest.TestingSeed{}
-	s.SetupAssertSigning("canonical", s)
+	s.SeedSnaps = &seedtest.SeedSnaps{}
+	s.SetupAssertSigning("canonical")
 
 	dir := c.MkDir()
-
-	s.SnapsDir = filepath.Join(dir, "snaps")
```

Comment:
so since none of the tests in this file use the snaps dir, it's safe to never create this dir, correct?